cmd/deploy/command_configuration.go

```diff
@@ -46,6 +46,12 @@ func setDeployOptionsValuesFromManifest(ctx context.Context, deployOptions *Opti
 		deployOptions.Manifest.Namespace = okteto.Context().Namespace
 	}
 
+	remote := utils.LoadBoolean(constants.OktetoDeployRemote)
+
+	if !remote || deployOptions.Manifest.Deploy.Remote == true {
```

Comment:
We need to check if `deployOptions.Manifest` and `deployOptions.Manifest.Deploy` could be nil, otherwise we'd be throwing a panic

Reply:
In both the cases, should we add a panic for _ErrNoInferManifestFound_?

Reply:
use remote deployer vs local deployer seems to be distributed among `command_configuration.go` and `GetDeployer()`. Could we unify it in a function in `deploy.go` file? WDYT? cc @nitishfy @jLopezbarb

Reply:
Done!